aggregators/omnibox-is-themed/index.js

```diff
@@ -19,7 +19,7 @@
 
 const Aggregate = require('../aggregate');
 
-class AddToHomescreen extends Aggregate {
+class OmniboxThemeColor extends Aggregate {
```

Comment:
I think this is fixing a mistake from when Paul split up the old `AddToHomescreen`.

On master, both `aggregators/omnibox-is-themed/index.js` and `aggregators/will-get-add-to-homescreen-prompt/index.js` contain a class named `AddToHomescreen`